tests/ip_config_test.py

```diff
@@ -177,6 +177,31 @@ def test_remove_interface_case_sensitive_mock_ipv6_w_apply_patch(self):
         print(result.exit_code, result.output)
         assert "converted ipv6 address to lowercase fc00::1~1126 with prefix /INTERFACE/Ethernet12| in value: /INTERFACE/Ethernet12|FC00::1~1126" in result.output
 
+    def test_intf_vrf_bind_unbind(self):
+        runner = CliRunner()
+        db = Db()
+        obj = {'config_db':db.cfgdb, 'namespace':db.db.namespace}
+
+        result = runner.invoke(config.config.commands["interface"].commands["vrf"].commands["bind"], ["Ethernet64", "Vrf1"], obj=obj)
+        print(result.exit_code, result.output)
+        assert result.exit_code == 0
+
+        result = runner.invoke(config.config.commands["interface"].commands["vrf"].commands["unbind"], ["Ethernet64"], obj=obj)
+        print(result.exit_code, result.output)
+        assert result.exit_code == 0
+
+    def test_intf_vrf_bind_unbind(self):
```

Comment:
I think we can remove this duplicate testcase.

Reply:
Fixed, dupe by copy past